checker/raw_result.go

```diff
@@ -111,6 +111,7 @@ const (
 // PinningDependenciesData represents pinned dependency data.
 type PinningDependenciesData struct {
 	Dependencies []Dependency
+	Incomplete   []error // jobs or files with errors may have incomplete results
```

Comment:
Two questions, since these structs are intended to be flexible to avoid breakages:
1. Is `Incomplete` better than `Skipped` or `ProcessingErrors` or something like that?
2. Do you intend this to be final? Since the type is defined as a slice of errors, we can't necessarily expand it later. (until the next major version)